frontend/src/features/public-form/components/FormFields/PublicRespondentEmailField.tsx

```diff
@@ -0,0 +1,71 @@
+import { Controller, RegisterOptions, useFormContext } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { Box, FormControl, FormLabel, Text } from '@chakra-ui/react'
+import { get } from 'lodash'
+import isEmail from 'validator/lib/isEmail'
+
+import { RESPONDENT_EMAIL_FIELD_ID } from '~shared/constants'
+
+import { useOptionalAdminEmailValidationRules } from '~utils/formValidation'
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import { TagInput } from '~components/TagInput'
+
+export const PublicRespondentEmailField = (): JSX.Element => {
+  const { t } = useTranslation()
+
+  const baseRules = useOptionalAdminEmailValidationRules()
+  const rules: RegisterOptions<
+    { respondent_email_field: string[] },
+    'respondent_email_field'
+  > = {
+    ...baseRules,
+    validate: {
+      ...baseRules.validate,
+      maxEmails: (value: string[]) =>
+        value.length <= 5 || 'Please enter a maximum of 5 emails',
+    },
+  }
+
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext<{ respondent_email_field: string[] }>()
+
+  return (
+    <FormControl isInvalid={!!errors?.[RESPONDENT_EMAIL_FIELD_ID]}>
```

Comment:
Related to the below,  since the `isInvalid` here will evaluate to truthy value when `get(errors, `${RESPONDENT_EMAIL_FIELD_ID}.message`)` exists, thus the FormResponseError will show up without need for the conditional check with ternary operator.